controllers/servers/host_definer/types.py

```diff
@@ -5,14 +5,18 @@
 @dataclass
 class DefineHostRequest:
     prefix: str = ''
-    connectivity_type: str = ''
-    node_id: str = ''
+    connectivity_type_from_user: str = ''
+    node_id_from_host_definition: str = ''
+    node_id_from_csi_node: str = ''
     array_connection_info: ArrayConnectionInfo = ArrayConnectionInfo(array_addresses='', user='', password='')
 
 
 @dataclass
 class DefineHostResponse:
     error_message: str = ''
+    connectivity_type: str = ''
+    ports: list = field(default_factory=list)
+    node_name_on_storage: str = ''
```

Comment:
is there a real need to define a default value? are they all can be not initiated?

Reply:
can I open a ticket for it and for all the other dataclasses in the hostDefinition?

Reply:
sure, put here ticket num

Reply:
CSI-4868